packages/forms-web-app/src/controllers/common/enter-code.js

```diff
@@ -107,81 +101,112 @@ async function sendTokenToLpaUser(req) {
 	}
 }
 
-const getEnterCode = (views, appealInSession) => {
+/**
+ * @param {{EMAIL_ADDRESS: string, ENTER_CODE: string, REQUEST_NEW_CODE: string}} views
+ * @param {boolean} [isGeneralLogin]
```

Comment:
Could we explain the meaning of this a little? Also wondering if an options object with that as a property would be clearer in usage: `getEnterCode(views, {isGeneralLogin: true})` ?